cookbooks/aws-parallelcluster-slurm/recipes/update_head_node.rb

```diff
@@ -176,7 +176,7 @@ def update_nodes_in_queue(strategy, queues)
 replace_or_add "Update compute console logging" do
   path "/etc/parallelcluster/slurm_plugin/parallelcluster_clustermgtd.conf"
   pattern "compute_console_logging_enabled*"
-  line "compute_console_logging_enabled = #{node['cluster']['config'].dig(:Monitoring, :Logs, :CloudWatch, :Enabled) || node['cluster']['slurm_console_logging']['enabled']}"
+  line "compute_console_logging_enabled = <%= node['cluster']['cw_logging_enabled'] == 'true' && node['cluster']['slurm_console_logging']['enabled'] %>"
```

Comment:
For better readability and consistency with the rest of the code, I believe we should use a `lazy {}` construct with a `not_if` condition on the whole `replace_or_add` block. Something similar to what we do [here](https://github.com/aws/aws-parallelcluster-cookbook/blob/5377d823e239ebb6537d68f907ea699fdb97618c/cookbooks/aws-parallelcluster-config/recipes/update_shared_storages.rb#L203-L211).

What do you think?

Reply:
I've removed the `['cluster']['slurm_console_logging']['enabled']` altogether since it wasn't really serving any useful purpose and just simplified to use the value provided by `['cluster']['cw_logging_enabled']`.

Reply:
I've removed this entire resource in the last commit.